### DISCUSSION TITLE
Feature to Mention a username in Discussions.

### DISCUSSION THREAD

**Discussion by shubhxg** [dailydotdev/daily#812] [Feature Requests]
If the Ability to Mention a username in Discussions is added then it will remove the confusion of discussion and we can reply to the particular user in the discussion.

Added a screenshot this is what it looks like right now:

![2023-02-08 17_40_51-Most discussed posts on daily dev - Brave](https://user-images.githubusercontent.com/69891912/217526547-a1cc317c-e24b-4a94-b306-95abc05d5cb7.png)

After the feature is added:

![2023-02-08 17_40_51-Most discussed posts on daily dev - Brave](https://user-images.githubusercontent.com/69891912/217527548-92fa799f-5867-4bb0-bc24-33a680035c9c.png)

**Comment by idoshamun** [comment]
Hi @shubhsharma19,

You can already mention users in the discussions. Type the user handle with a leading "@" and it should mention them. e.g "@shubhsharma19"

**Reply by shubhxg** [reply]
Ok thankyou :)

**Comment by nimrodkra** [comment]
@shubhsharma19 Update: we now automatically add a mention of the user you reply to when you hit the "Reply" button. 
We can mark this as done!
